### PR TITLE
feat: collapse flux

### PR DESCRIPTION
**Pull request by drdelambre**
no idea what to call this.. but if you ever miss data explorer and want to use notebooks as a flux generator... knock on wood you know

`influx.set('collapseotron', true)` to start the sad waltz

Peek 2021-04-08 15-21

### CODE REVIEW

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1152)

File: src/flows/components/panel/InsertCellButton.tsx [hunk 1]
```diff
@@ -53,6 +99,29 @@ const InsertCellButton: FC<Props> = ({id}) => {
...
         active={popoverVisible.current}
       />
+      <FeatureFlag name="collapseotron">
+        <ClickOutside onClickOutside={clickOutside}>
+          <div className={collapseClassnames}>
+            {confirming && (
+              <div className="flow-divider--collapse-notice">
+                this will merge all previous notebook panels into a single flux
+                panel.
+                <span>there is no undo button.</span>
```
Comment (Asalem1) [inline src/flows/components/panel/InsertCellButton.tsx:109]:
```suggestion
                <span>there is no spoon.</span>
```

**Review by Asalem1 — APPROVED** [review]
Looks good to me, but why not just have a preview panel of the flux that's generated rather than merging the existing queries into one?

**Comment by drdelambre** [comment]
yeah.. design put forth that option, but it always begged the question; then what do you do with it? copy it from one place to paste it in another? export as task or dashboard buttons?

if they want to do that... they can totally still do that

but if we collapse the panels, at least they can stay within the same editing experience and potentially set the query to run on a schedule or something

File: src/flows/components/panel/InsertCellButton.tsx (hunk 1 above)
Comment (drdelambre) [reply src/flows/components/panel/InsertCellButton.tsx:109]: image

**Pull request merged**